pkg/adapter/sinker.go

```diff
@@ -33,8 +33,23 @@ func (s *sinker) processEventPayload(ctx context.Context, request *http.Request)
 		return err
 	}
 
-	// set logger with sha and event type
-	s.logger = s.logger.With("event-sha", s.event.SHA, "event-type", s.event.EventType)
+	// Enhanced structured logging with source repository context for operators
+	logFields := []interface{}{
+		"event-sha", s.event.SHA,
+		"event-type", s.event.EventType,
+		"source-repo-url", s.event.URL,
+	}
```

Comment:
```suggestion
	logFields := []interface{}{
		"event-sha", s.event.SHA,
		"event-type", s.event.EventType,
		"event", s.event.TriggerType,
		"source-repo-url", s.event.URL,
	}
```

in JIRA description, it is reported that "Event type (this is not always consistent)" I think its because for example when a label is added to a PR event is `pull_request` but event-type is `label-update` and that may be confusing so having `TriggerType` as event worth it.
cc: @chmouel